spotbugs/src/main/java/edu/umd/cs/findbugs/util/Util.java

```diff
@@ -338,7 +341,7 @@ public static String getFileExtensionIgnoringGz(File f) {
 
     public static boolean isPowerOfTwo(int i) {
         return i > 0
-            && (i | (i - 1)) + 1 == 2 * i;
```

Comment:
I don't really agree with the extra indentation here, but eh, if it means we can apply consistent formatting, it's a win.

Reply:
> I don't really agree with the extra indentation here
I think so, but i seem that extra indentations is contained in code base.
so, I will merge for consistent formatting.